bitnami/mediawiki/templates/metrics-svc.yaml

```diff
@@ -0,0 +1,29 @@
+{{- if .Values.metrics.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ printf "%s-metrics" (include "common.names.fullname" .) }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: metrics
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if or .Values.metrics.podAnnotations.annotations .Values.commonAnnotations }}
+  annotations:
+    {{- if .Values.metrics.podAnnotations.annotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.metrics.podAnnotations.annotations "context" $ ) | nindent 4 }}
+    {{- end }}
+    {{- if .Values.commonAnnotations }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- end }}
+spec:
+  type: ClusterIP
+  ports:
+    - name: metrics
+      port: 9117
```

Comment:
Could you set up the port number using the values.yaml, please?

Ex: https://github.com/bitnami/charts/blob/75ac318b1b8c902f16dd2b749d83e6b0904e6313/bitnami/wordpress/templates/metrics-svc.yaml#L25

something like .Values.metrics.port